src/networks.json

```diff
@@ -1463,7 +1463,26 @@
     "explorer": {
       "url": "https://testnet.airdao.io/explorer"
     },
-    "start": 3368046,
+    "start": 709476,
+    "logo": "ipfs://QmSxXjvWng3Diz4YwXDV2VqSPgMyzLYBNfkjJcr7rzkxom"
+  },
+  "30746": {
+    "key": "30746",
+    "name": "AirDAO Devnet",
+    "chainId": 30746,
+    "network": "devnet",
+    "testnet": true,
+    "multicall": "0x55Be7dd776fAbe89d93bAC66ed1cf0Ab31bdd6eB",
+    "rpc": [
+      "https://network-archive.ambrosus-dev.io"
+    ],
+    "ws": [
+      "wss://network-archive.ambrosus-dev.io/ws"
+    ],
+    "explorer": {
+      "url": "https://devnet.airdao.io/explorer"
```

Comment:
this link is not working https://devnet.airdao.io/explorer

Reply:
Yea, devnet explorer is down for now. Is it really required for dev net?

Reply:
Yep, need to verify the multicall contract code, start block etc